src/lwc/bdiFieldMappingModal/bdiFieldMappingModal.js

```diff
@@ -212,23 +211,14 @@ export default class bdiFieldMappingModal extends LightningElement {
             this.mappedDiFieldDescribes = event.mappedDiFieldDescribes;
             this.targetObjectFieldDescribes = event.targetObjectFieldDescribes;
 
-            this.row = event.row;
-
-            if (this.row) {
-                // Edit row
+            if (event.row) {
+                // Edit field mapping
                 this.modalMode = 'edit';
-                this.selectedSourceFieldLabel = this.row.Source_Field_Label;
```

Comment:
Just want to make sure I'm reading the intent here - instead of copying the field values over like was happening in this block of removed code, you are parsing the results into the fields? I think I like the switch, if that's the case!